Makefile

```diff
@@ -123,7 +123,7 @@ install:
 
 .PHONY: test
 test: verify_version $(THRIFTRW)
-	PATH=$(_GENERATE_DEPS_DIR):$$PATH go test $(PACKAGES)
+	PATH=$(_GENERATE_DEPS_DIR):$$PATH go test -run . -bench . $(PACKAGES)
```

Comment:
Should we make this a separate target? If we're going to have stress tests later, this may mess with CI.